101-vsts-cloudloadtest-rig/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "VSTS-CloudLoadTest-SelfProvisionedRig",
```

Comment:
Please use a descriptive itemDisplayName as this shows up on Azure.com. This seems like a folder name. See example of good name https://github.com/Azure/azure-quickstart-templates/blob/master/301-drupal8-vmss-glusterfs-mysql/metadata.json#L2